lighthouse-core/report/html/renderer/util.js

```diff
@@ -401,6 +401,21 @@ class Util {
     };
   }
 
+  /**
+   * @param {string} sourceString
+   * @return {string}
+   */
+  static str(sourceString) {
```

Comment:
does it have to be `str`? :)

I get that we don't want it to be too similar to i18n ones since it's quite different, but maybe a bit more descriptive since we're already forced to wrap :D